packages/docusaurus-theme-search-algolia/src/theme/hooks/useAlgoliaContextualFacetFilters.ts

```diff
@@ -6,9 +6,10 @@
  */
 
 import useContextualSearchFilters from '@theme/hooks/useContextualSearchFilters';
```

Comment:
i'm unsure if this is correct, we are importing this hook from theme-classic and thus compilation doesn't work as this package doesn't rely on theme